javascript-carousel/main.js

```diff
@@ -0,0 +1,67 @@
+const imagesArray = ['images/001.png', 'images/004.png', 'images/007.png', 'images/025.png', 'images/039.png'];
+let currentIndex = 0;
+
+const $leftArrow = document.querySelector('#left-arrow');
+const $rightArrow = document.querySelector('#right-arrow');
+const $img = document.querySelector('#image');
+const $firstCircle = document.querySelector('#first-circle');
+const $secondCircle = document.querySelector('#second-circle');
+const $thirdCircle = document.querySelector('#third-circle');
+const $fourthCircle = document.querySelector('#fourth-circle');
+const $fifthCircle = document.querySelector('#fifth-circle');
+
+$leftArrow.addEventListener('click', event => {
+  if (currentIndex === 0) {
+    currentIndex = 4;
+    $img.setAttribute('src', imagesArray[currentIndex]);
+  } else {
+    currentIndex--;
+    $img.setAttribute('src', imagesArray[currentIndex]);
+  }
+  updateCircle();
+});
+
+$rightArrow.addEventListener('click', event => {
+  if (currentIndex === 4) {
+    currentIndex = 0;
```

Comment:
Calculate your bounds based on the size of the image array so that if you change the image array, you don't have to modify your code.